packages/python-packages/protocol-stub-generator/tox.ini

```diff
@@ -0,0 +1,11 @@
+[tox]
+# note that this envlist is the default set of environments that will run if a target environment is not selected.
+envlist = test
+
+[testenv]
+deps = pytest
+ignore_args=--ignore=.tox --ignore=build --ignore=.eggs
+default_pytest_params = --junitxml={toxinidir}/test-junit-{envname}.xml --verbose --durations=10
+commands = 
+    pytest {toxinidir}/tests
```

Comment:
do you really have tests now or is this just a placeholder?

Reply:
placeholder